bin/heudiconv

```diff
@@ -1,19 +1,25 @@
 #!/usr/bin/env python
 
-"""Convert DICOM TimTrio dirs based on heuristic info
+"""Convert DICOM dirs based on heuristic info
 
-This function uses DicomStack and mri_convert to convert Siemens
-TrioTim directories. It proceeds by extracting dicominfo from each
-subject and writing a config file $subject_id/$subject_id.auto.txt in
-the output directory. Users can create a copy of the file called
-$subject_id.edit.txt and modify it to change the files that are
-converted. This edited file will always overwrite the original file. If
-there is a need to revert to original state, please delete this edit.txt
-file and rerun the conversion
+This script uses DicomStack and mri_convert to convert DICOM directories.
```

Comment:
took your version and expanded... most probably will need to resolve conflicts with master by now ;) thanks